README.md

```diff
@@ -1,4 +1,31 @@
 # PlaceRecognitionDB
 PlaceRecognitionDB is a tool for creating optimally sized databases (containing the minimum number of frames covering the scene) for place recognition task from RGBD and LiDAR data.
 
-The goal of the project is to create a tool with an optimal compression strategy for any RGBD and LiDAR data.
+The tool supports several basic methods for creating databases, as well as the DominatingSet method for creating optimal databases.
+
+## Datasets format
+To use the tool, your data must be in a specific format.
+* Color images in any format.
+* Depth images corresponding to color images in 16-bit grayscale format or point clouds in `.pcd` format.
+* The trajectory containing one pose in each line in `timestamp (optional) tx ty tz qx qy qz qw` format.
```

Comment:
What about depth scale and camera intrinsics? It is better to at least specify that they are also required.

Reply:
And as a discussion, maybe it is better to read intrinsics and depth scale from a file since this is something that is fixed in a dataset and specifying it manually each time seems redundant?